components/movies.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const axios = require("axios");
+require('dotenv').config();
+
+const MOVIE_API_KEY = process.env.MOVIE_API_KEY;
+
+class Films{
```

Comment:
Great work Steve!

Feedback: Rename class to `Movie` 👍

Reply:
Oh also, I noticed this sits inside of  a folder named `components`. I would consider renaming this folder to `modules` since we have components on the frontend in React 👍